neo/Network/P2P/Payloads/Transaction.cs

```diff
@@ -157,6 +157,23 @@ public JObject ToJson()
             return json;
         }
 
+        public static Transaction FromJson(JObject json)
```

Comment:
I think the easiest way to recover `Transaction` from JSON RPC repsonse is to set `verbose` to `false`.

https://github.com/neo-project/neo/blob/897df9c78d97e44f4526a7cdcb63bb2b08ebcbbf/neo/Network/RPC/RpcServer.cs#L173

Then you can easily get the `Transaction` by deserializing from the hex.

The block is the same.

Reply:
Hi Erik,
The json constains more information when **verbose** is **true**
![image](https://user-images.githubusercontent.com/4393234/60228302-d836e900-98c4-11e9-9e92-e998d4afc49a.png)

Reply:
Yes. But in SDK, you just receive it and convert it into a `Transaction`. So there's no difference.

Reply:
@erikzhang dalao, so we only need to provide the GetRawTransaction() method with **verbose** set to **false** inside, and let the developers using this method to decide if they need to deserialize **Transaction** from the HEX, am I understanding this right?
![image](https://user-images.githubusercontent.com/33386299/60330991-3ac3de00-99c6-11e9-9aa5-0469a3304ae8.png)
Or should I provide another GetRawTransaction() method to deserialize the HEX like following code?
![image](https://user-images.githubusercontent.com/33386299/60334685-a5791780-99ce-11e9-864c-6247cd1f2b80.png)

Reply:
We have wrapped the Transaction to SDK_Transaction to contain more infotrmation, or we can just keep one method to get Transaction:
![image](https://user-images.githubusercontent.com/4393234/60407596-f9157c00-9bed-11e9-9521-323e8fb1c10c.png)

Reply:
This SDK is intended to be used mainly by code?

Reply:
Yes, this is an SDK for developers.